kmip/pie/objects.py

```diff
@@ -599,3 +599,94 @@ def __ne__(self, other):
             return not (self == other)
         else:
             return NotImplemented
+
+
+class OpaqueObject(ManagedObject):
+    """
+    The OpaqueObject class of the simplified KMIP object hierarchy.
+
+    OpaqueObject is one of several ManagedObjects and is one of the core KMIP
+    objects that are the subject of key management operations. For more
+    information, see Section 2.2 of the KMIP 1.1 specification.
+
+    Attributes:
+        opaque_type: The type of the opaque value.
+    """
+
+    def __init__(self, value, opaque_type, name='Opaque Object'):
```

Comment:
Yeah, it doesn't really make sense for opaque objects. It's opaque, it's just bytes, it shouldn't have a type.